lib/usageParams.js

```diff
@@ -17,26 +17,16 @@ const opts = yargs
     demand: true
   })
   .option('content-file', {
-    describe: 'json file that contains data to be import to your space',
+    describe: 'JSON file that contains data to be import to your space',
     type: 'string',
     demand: true
   })
   .option('skip-content-model', {
-    describe: 'Skips content types and locales. Copies only entries and assets',
```

Comment:
Cus it would still import other stuff like webhooks etc. So `only entries and assets` would be wrong.